src/ripple/protocol/Feature.h

```diff
@@ -349,6 +349,7 @@ extern uint256 const fixNFTokenRemint;
 extern uint256 const fixReducedOffersV1;
 extern uint256 const featureClawback;
 extern uint256 const featureXChainBridge;
+extern uint256 const fixPayChanV1;
```

Comment:
nit: IMO this should have a more descriptive name, like `fixPayChanCancelAfter` or something - especially since it's one specific fix and not an assortment of paychan fixes.